syz-verifier/main.go

```diff
@@ -464,17 +464,13 @@ func createReport(rr *verf.ResultReport, pools int) []byte {
 
 		// Ensure results are ordered by pool index.
 		for i := 0; i < pools; i++ {
-			errno, ok := cr.Errnos[i]
+			state, ok := cr.States[i]
 			if !ok {
 				// VM crashed so we don't have reports from this pool.
 				continue
 			}
 
-			errnoDesc := "success"
-			if errno != 0 {
-				errnoDesc = syscall.Errno(errno).Error()
-			}
-			data += fmt.Sprintf("\t↳ Pool: %d, Flag: %d, Errno: %d (%s)\n", i, cr.Flags[i], errno, errnoDesc)
+			data += fmt.Sprintf("\t↳ Pool: %d, %s\n", i, state.String())
```

Comment:
You may remove `.String()`. String method is called automatically. That's how, say, error's are routinely printed.